### PR TITLE
Replace Python 2 headers with Python 3 headers

### LINKED ISSUE #81: Python 2 functionality is broken
Example:

screen shot 2018-04-06 at 2 27 54 pm

### PR DESCRIPTION
**Pull request by nkadonaga**
~~How do I test whether this works and which type of unit test should I add to confirm that I changed something? I'm not really sure what the "bug" I'm fixing is.~~ Added tests

Resolve #81 

- [x] unit tests pass
- [x] manually testing works

Before: 

screen shot 2018-04-06 at 2 27 54 pm

After:

screen shot 2018-04-06 at 2 28 15 pm

### CODE REVIEW

File: tests/test_interpreter.py [hunk 1]
```diff
@@ -140,6 +140,8 @@ def test_print(self):
     def test_factorial(self):
         response = self.get_code_response(FACTORIAL_CODE)
```
Comment (mjenrungrot) [inline tests/test_interpreter.py]: Can we use the same style as the code below? Something like this
```
self.get_code_response(code=FACTORIAL_CODE, args=TARGET_36_ARG
```

File: app/my_app.py [hunk 2]
```diff
@@ -7,6 +7,7 @@
 from flask import request, jsonify
 from coconut.convenience import parse, setup
 from coconut.exceptions import CoconutException
+from coconut.compiler.header import getheader, section
 from flask_cors import CORS
```
Comment (mjenrungrot) [inline app/my_app.py]: Can we reorder the import portion?
```
import json
import os
import sys
import contextlib
import traceback
from io import StringIO

from coconut.convenience import parse, setup
from coconut.exceptions import CoconutException
from coconut.compiler.header import getheader, section
from flask import request, jsonify
from flask_cors import CORS

from app import create_app
from .trace import extract_trace_py, extract_trace_coco
```

File: app/my_app.py [hunk 3]
```diff
@@ -59,21 +60,25 @@ def coconut():
     if not compile_error:
         print("Finish compilation")
 
-        SEPARATOR = "# Compiled Coconut: -----------------------------------------------------------\n\n"
+        SEPARATOR = section("Compiled Coconut")
```
Comment (mjenrungrot) [inline app/my_app.py]: Can you document the behavior of this external library code?

File: tests/test_interpreter.py (hunk 1 above)
Comment (mjenrungrot) [reply tests/test_interpreter.py]: Can we also test the behavior of the apps with `args=TARGET_27_ARG` on all of the tests?

Comment (nkadonaga) [reply tests/test_interpreter.py]: Done: added `args=TARGET_27_ARG` to all tests except `async_def`

**Pull request merged**